.chloggen/config.yaml

```diff
@@ -12,6 +12,7 @@ components:
     - cmd/mdatagen
     - connector/forward
     - connector/sample
+    - consumer/consumererror/xconsumererror
```

Comment:
i might be missing something. but this pr includes the changes to ad `config/config*` in a few places, but not in the chloggen config. is this expected?

Reply:
This was what happened when I ran `make gengithub`. Those folders have `metadata.yaml` files, they were added in https://github.com/open-telemetry/opentelemetry-collector/pull/14029. The `make gengithub` target must have been missed or something in this PR.